crates/engine/local/Cargo.toml

```diff
@@ -37,7 +37,7 @@ futures-util.workspace = true
 
 # misc
 eyre.workspace = true
-tracing.workspace = true
+tracing = { workspace = true, features = ["std"] }
```

Comment:
```suggestion
tracing.workspace = true
```

Reply:
yes, I'll do this change but can you send me under which crate folder I have to make this change, so change their revert cargo.toml. @mattsse  @emhane

Reply:
all of them